components/calendar/src/types.rs

```diff
@@ -92,11 +92,7 @@ pub struct EraYear {
     pub era: TinyStr16,
     /// An era index, for calendars with a small set of eras.
     ///
-    /// It is obtained by ordering all eras' start/end dates, and for eras sharing a date, put the negative
-    /// one first. For example, bce < ce.
-    ///
-    /// As eras might be added or removed across CLDR releases, this does not have a stable correspondence
-    /// with the era code.
+    /// The only guarantee we make is that these values are stable, even under CLDR code changes.
     pub era_index: Option<u8>,
```

Comment:
Nit (Optional): Don't change these docs until you actually change the numbers. I read this and thought this PR was changing the indices to conform to our new requirement, but it is just a refactoring to change the source of truth.

Reply:
well it's amibguous what "all eras' start/end dates" means across CLDR releases, so I don't think the current docs are accurate